source/features/repo-age.tsx

```diff
@@ -43,12 +43,20 @@ const getRepoAge = async (commitSha: string, commitsCount: number): Promise<[str
 };
 
 const getFirstCommit = cache.function(async (): Promise<[string, string] | undefined> => {
-	const commitInfo = await elementReady<HTMLAnchorElement | HTMLScriptElement>('.commit-tease a[href*="/commit/"], include-fragment.commit-tease');
+	const commitInfo = await elementReady<HTMLAnchorElement | HTMLScriptElement>([
+		// "Repository refresh" layout
+		'.Box-header--blue .hx_avatar_stack_commit + div a[href*="/commit/"]',
```

Comment:
Can this selector be improved/shortened?

Reply:
It could be shortened like this:

```suggestion
		'.Box-header--blue a[href*="/commit/"]',
```

But since `.Box-header--blue` is quite generic (and I don't know if it is used elsewhere), I opted for more specific selector.